DependencyInjection/Configuration.php

```diff
@@ -33,6 +33,9 @@ public function getConfigTreeBuilder()
             ->children()
                 ->scalarNode('request_time_log_level')->defaultValue(LogLevel::DEBUG)->end()
             ->end()
+            ->children()
+                ->booleanNode('ignore_request_body_on_get')->defaultValue(false)->end()
```

Comment:
I would suggest something more "unified" like `strict_mode`.

`ignore_request_body_on_get` mentions only `GET`, but in reality also in `HEAD`, `OPTIONS`, `TRACE`.

Could you also add some documentation in README.

Reply:
Fixed